src/status_im/ui/screens/wallet/components.cljs

```diff
@@ -27,10 +27,11 @@
 (def default-action (actions/back-white actions/default-handler))
 
 (defn- toolbar
-  ([title] (toolbar default-action title))
-  ([action title] (toolbar action title nil))
-  ([action title options]
-   [toolbar/toolbar {:style styles/toolbar}
+  ([title] (toolbar nil title))
+  ([style title] (toolbar style default-action title))
+  ([style action title] (toolbar style action title nil))
```

Comment:
You change contract for arity 2 and 3. How does that work for all toolbars using those arities?

Reply:
I thought taking style or props as a first argument would be better. This component is used only in wallet section, and currently there's no source code using arity 2 or 3 of this component. So I think this change will not break anything. Is there something I'm missing?